12-cleanup.md

```diff
@@ -23,12 +23,18 @@ After you are done exploring your deployed [AKS secure baseline cluster](./), yo
    > Because this reference implementation enables soft delete on Key Vault, execute a purge so your next deployment of this implementation doesn't run into a naming conflict.
 
    ```bash
-   az keyvault purge -n $KEYVAULT_NAME
+   az keyvault purge -n $KEYVAULT_NAME_BU0001A0042_03
+   az keyvault purge -n $KEYVAULT_NAME_BU0001A0042_04
    ```
 
-1. If any temporary changes were made to Azure AD or Azure RBAC permissions consider removing those as well.
+1. Delete your Policy Assignments
```

Comment:
Combine with previous text so users can go to portal if they want. 

add some context/comments on what we are doing with the following scripts.

Reply:
done | addressed from [540a7](https://github.com/mspnp/aks-secure-baseline/pull/155/commits/540a7de96d222e36124ac9a39d3f48b2003ef247)